roles/engineblock/tasks/install-branch.yml

```diff
@@ -37,3 +37,4 @@
   notify:
     - "restart httpd"
     - "restart php-fpm"
```

Comment:
We don't need "restart php-fpm" anymore. The app/console commands are also php72 only